<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeEditor</class>
 <widget class="QDialog" name="CodeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>632</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
        background: #39464d;
}

QMessageBox {
        background: rgb(220, 220, 220);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0,15,0,4,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,3,4,3">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QComboBox" name="languageSelect">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>104</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
padding: 1px 23px 1px 3px;
min-width: 6em;
color: #ffffff;
border: none;

</string>
           </property>
           <item>
            <property name="text">
             <string>Programming language</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C++</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C#</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Java</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Python</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="loaderButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
   background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
   border-color: #545454;
   border-width: 1px;
   border-radius: 5px;
   border-style: solid;
   color: white;
                     }
QPushButton:pressed {
   background: qradialgradient(cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 #dddddd, stop:1 #777777);
}</string>
           </property>
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="flagLine">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
border: 1px solid #333333;
        border-radius: 5px;
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
        min-width: 6em;
        color: #ffffff;

padding: 0px;
text-align: center;
}</string>
           </property>
           <property name="placeholderText">
            <string>Insert flags</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="submitCodeButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft Sans Serif</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
   background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
   border-color: #545454;
   border-width: 1px;
   border-radius: 5px;
   border-style: solid;
   color: white;
                     }
QPushButton:pressed {
   background: qradialgradient(cx:0.5, cy:0.5, radius:1, fx:0.5, fy:0.5, stop:0 #dddddd, stop:1 #777777);
}</string>
           </property>
           <property name="text">
            <string>Open code file</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>6</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>106</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333333;
border-top-left-radius: 5px;
border-top-right-radius: 5px;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
padding: 1px 23px 1px 3px;
min-width: 6em;
color: #ffffff;
text-align: center;</string>
       </property>
       <property name="text">
        <string>Editor</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Editor" name="codeEditor">
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit {
background: white;
}</string>
       </property>
       <property name="tabStopWidth">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="loaderLabel">
       <property name="minimumSize">
        <size>
         <width>106</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft Sans Serif</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #333333;
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #797979, stop:0.48 #696969, stop:0.52 #5e5e5e, stop:1 #4f4f4f);
padding: 1px 23px 1px 3px;
min-width: 6em;
color: #ffffff;
text-align: center;</string>
       </property>
       <property name="text">
        <string>Output</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="loaderOutput">
       <property name="font">
        <font>
         <family>Courier</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPlainTextEdit {
background: white;
}</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="plainText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>2</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Editor</class>
   <extends>QPlainTextEdit</extends>
   <header>editor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
